plugins/woocommerce/tests/e2e-pw/tests/shopper/shop-products-filter-by-price.spec.js

```diff
@@ -0,0 +1,180 @@
+const { test, expect } = require( '@playwright/test' );
+const { admin } = require( '../../test-data/data' );
+const { closeWelcomeModal } = require( '../../utils/editor' );
+const wcApi = require( '@woocommerce/woocommerce-rest-api' ).default;
+
+const singleProductPrice1 = '10';
+const singleProductPrice2 = '50';
+const singleProductPrice3 = '200';
+
+const simpleProductName = 'AAA Filter Products';
+
+const productsFilteringPageTitle = 'Products Filtering';
+const productsFilteringPageSlug = productsFilteringPageTitle
+	.replace( / /gi, '-' )
+	.toLowerCase();
+
+let product1Id, product2Id, product3Id;
+
+test.describe( 'Filter items in the shop by product price', () => {
+	test.beforeAll( async ( { baseURL } ) => {
+		const api = new wcApi( {
+			url: baseURL,
+			consumerKey: process.env.CONSUMER_KEY,
+			consumerSecret: process.env.CONSUMER_SECRET,
+			version: 'wc/v3',
+		} );
+		// add products
+		await api
+			.post( 'products', {
+				name: simpleProductName + ' 1',
+				type: 'simple',
+				regular_price: singleProductPrice1,
+			} )
+			.then( ( response ) => {
+				product1Id = response.data.id;
+			} );
+		await api
+			.post( 'products', {
+				name: simpleProductName + ' 2',
+				type: 'simple',
+				regular_price: singleProductPrice2,
+			} )
+			.then( ( response ) => {
+				product2Id = response.data.id;
+			} );
+		await api
+			.post( 'products', {
+				name: simpleProductName + ' 3',
+				type: 'simple',
+				regular_price: singleProductPrice3,
+			} )
+			.then( ( response ) => {
+				product3Id = response.data.id;
+			} );
+	} );
+
+	test.afterAll( async ( { baseURL } ) => {
+		const api = new wcApi( {
+			url: baseURL,
+			consumerKey: process.env.CONSUMER_KEY,
+			consumerSecret: process.env.CONSUMER_SECRET,
+			version: 'wc/v3',
+		} );
+		await api.post( 'products/batch', {
+			delete: [ product1Id, product2Id, product3Id ],
+		} );
+	} );
+
+	test( 'can create a new page with products filter and all products', async ( {
+		page,
+	} ) => {
+		await page.goto( 'wp-admin/post-new.php?post_type=page' );
+		await page.waitForLoadState( 'networkidle' );
```

Comment:
Using `networkidle` is discouraged, eslint reports it as an error. You should asses readiness in other ways. In fact, the next line already checks for readiness before using the element, so this `networkidle` is not needed anyway.